src/webviews/components/settingsPanel.vue

```diff
@@ -8,6 +8,9 @@
                 type="checkbox"
                 v-if="collapseable || startCollapsed"
                 v-model="collapsed"
+                :aria-expanded="!collapsed"
```

Comment:
Adding ARIA attributes is very important for testing since the interfaces are based around accessibility.